crypto/store/store_result.c

```diff
@@ -504,8 +494,12 @@ static int try_crl(struct extracted_param_data_st *data, OSSL_STORE_INFO **v,
         || data->object_type == OSSL_OBJECT_CRL) {
         X509_CRL *crl;
 
+        /* Discard DER decoding errors, it only means we return "empty handed" */
+        ERR_set_mark();
```

Comment:
Similarly to the above X509 case I would not clear the errors if the data->object_type is set.